src/globals.ts

```diff
@@ -8,5 +8,21 @@ export interface PageTitleAndLink {
   link: string;
 }
 
+export interface AvailableCourses {
+  cursor: string;
+  summaries: Summary[];
+  more: boolean;
+}
+
+export interface Summary {
```

Comment:
Summary properties are what it is in JSON, we can't change those. Otherwise it will not be mapped properly.

@shihabmridha - `Your Comment was - Let's move these to globals.ts. Also, make Summary's properties camelcase.`